tendrl/gluster_integration/objects/volume/atoms/get_mapping/__init__.py

```diff
@@ -0,0 +1,111 @@
+import json
+
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from tendrl.commons import objects
+from tendrl.commons.objects.job import Job
+
+
+class GetMapping(objects.BaseAtom):
```

Comment:
Whats this to and fro folks

This Atom generates brick mapping right? then call it `GenerateBrickMapping`